### ISSUE TITLE
Forbidden / unauthorized message for unsuccessful auth

### ISSUE THREAD

**Issue by jmhooper** [18F/dolores-landingham-slack-bot#213]
Currently, if you attempt to login via Github, and you aren't a member of the appropriate organization or your organization membership is set to private the app renders an error message:

![image](https://cloud.githubusercontent.com/assets/963654/18556811/319653f4-7b32-11e6-8b69-09b7818b100f.png)

In production this will render a bank, white screen. This may be confusing for users who may not understand that they are not authenticated.

I think we should consider either:

- Redirecting to the sign in screen with a flash message saying there was a problem or
- Display a page describing the error along with some hints about what may have gone wrong.

I'm a fan of the latter and have put together a page for that already:

![image](https://cloud.githubusercontent.com/assets/963654/18556918/88eb7c7e-7b32-11e6-9b3e-95a41c0a4d37.png)

Looking for some feedback on this thought. If the team is in favor of an error page I can go ahead open a PR with what I've put together.

**Comment by jessieay** [comment]
I like this idea! Agree that error page is not helpful.

 I know someone created a landing page for signing in before, would make
sense to use the same layout for this page.

Thanks for bringing this up! We are always trying to improve the onboarding
experience for new devs and I think this will help them as much as it helps
end users.

**Comment by jmhooper** [comment]
@jessieay: I'm looking for the sign in landing page but I can't seem to find it.

I found a new session page: https://github.com/18F/dolores-landingham-slack-bot/blob/6e59a366ab90ee06a5fc2519c7c6686a0cc10b6f/app/views/sessions/new.html.erb

This one is kind of weird because it renders with the application layout so you get the sidebar even though you aren't authed:

![image](https://cloud.githubusercontent.com/assets/963654/18568920/4a374fa2-7b65-11e6-8ea6-8699aec6375a.png)

Maybe that's a new issue in the session controller where it should render outside the layout? Ref: https://github.com/18F/dolores-landingham-slack-bot/blob/6e59a366ab90ee06a5fc2519c7c6686a0cc10b6f/app/controllers/sessions_controller.rb#L4

I built out the error page to render in the application layout because the sign in page did, but they both may need to be changed.

I'll put up a PR with those changes.
